os-config_arm.yml

```diff
@@ -0,0 +1,328 @@
+hostname: rancher
```

Comment:
@luxas there is also a difference in that acpid is not included in ARM.  but again, nothing some sed can't fix.  I personally have never liked that the version has to be updated in this file.  One point you bring up though is should the arch be in the image name or label.  Right now it's in the label.  I think we went with that approach because it seemed more inline with say a dpkg versioning scheme where the arch is not in the package name.